stanford-earth/matson
# EARTH-514 Changed hero banner margin to only act on the first full wi…

# READY FOR REVIEW

# Summary
- Target only the first hero banner pattern that is not full width.

# Needed By (Date)
- Soon

# Urgency
- medium

# Steps to Test

1. Checkout this branch and same branch name of [stanford_components](https://github.com/SU-SWS/stanford_components/pull/91) and [stanford_complex_page](https://github.com/SU-SWS/stanford_complex_page/pull/5)
2. `drush cr; drush features-import stanford_complex_page -y`
3. create a complex page with a video paragraph item.
4. put it at the top with full width - make sure it gets pulled to the top of the page.
5. Put it at the top but not full width - make sure it is below the top banner portion
6. Put it in the middle of the page full width and not full width - make sure it doesnt get the negative top margin
7. review these in mobile. ( not sure what to do about top hero when in mobile.)
8. do happy dance.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_complex_page/pull/5
- https://github.com/SU-SWS/stanford_components/pull/91

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
